### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by AerGameChannel**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
-  "logChannel": null,
-  "action": "DELETE",
+  "token": "put your bot's token here",
+  "logChannel": "put log channel id here",
```
Comment (redpfire) [inline config.json:3]: I specifically set this to null so you could log into the same channel.

File: filter.json [hunk 2]
```diff
@@ -25,12 +25,5 @@
   "treadeoffer",
   "discordgifts",
   "discorcl",
-  "dlscord.\n",
```
Comment (redpfire) [inline filter.json:28]: Why would you delete that?? Your filter didn't work in my use case!!

File: config.json (hunk 1 above)
Comment (AerGameChannel) [reply config.json:3]: Doesn't it log into the same channel if it can't resolve the channel?

Comment (AerGameChannel) [reply config.json:3]: It does

image
image

Comment (redpfire) [reply config.json:3]: Yes, it does! That was the default behaviour in your previous case.

Comment (redpfire) [reply config.json:3]: Your approach also works, my was just a clean route.

**Pull request merged**